pkg/flexvolume/client.go

```diff
@@ -22,43 +39,119 @@ func (c *client) Init() error {
 }
 
 func (c *client) Attach(jsonOptions map[string]string) error {
+	var output []byte
 	_, err := c.apiClient.Attach(
 		context.Background(),
 		&AttachRequest{
 			JsonOptions: jsonOptions,
 		},
 	)
+	if err == nil {
+		a := AttachSuccessOutput{
+			Status: "Success",
+			Device: jsonOptions["volumeID"],
+		}
+		output, _ = json.Marshal(a)
+	} else {
+		a := FailureOutput{
+			Status:  "Failure",
+			Message: err.Error(),
+		}
+		output, _ = json.Marshal(a)
```

Comment:
I have completely removed json marshalling and replaced it with your suggestion of []byte success and failure constants.